docs/source/tutorials/building_blocks.rst

```diff
@@ -0,0 +1,247 @@
+           self.wait(1)
+
+.. image:: ../_static/building_blocks/addremove.gif
+    :align: center
+    :alt: a circle displayed for 1s then removed
+
+
+Placing mobjects
+================
+
+Let's define a new :class:`.Scene` called ``Shapes`` and :meth:`~.Scene.add`
+some mobjects to it.
+
+.. code-block:: python
+
+   class Shapes(Scene):
+       def construct(self):
+           circle = Circle()
+           square = Square()
+           triangle = Triangle()
+
+           circle.shift(LEFT)
+           square.shift(UP)
+           triangle.shift(RIGHT)
+
+           self.add(circle, square, triangle)
+           self.wait(1)
+
+This script generates a static picture that displays a circle, a square, and a
+triangle:
+
+.. image:: ../_static/building_blocks/shapes.png
+    :align: center
+    :alt: figure of a circle, a square, and a triangle
+
+By default, mobjects are placed at the center of coordinates, or *origin*, when
+they are first created.  They are also given some default colors.  Further, the
+``Shapes`` scene places the mobjects by using the :meth:`.shift` method.  The
+square is shifted one unit in the ``UP`` direction from the origin, while the
+circle and triangle are shifted one unit ``LEFT`` and ``RIGHT``, respectively.
+
+.. attention:: Unlike other graphics software, manim places the center of
+               coordinates at the center of the screen.  The positive vertical
+               direction is up, and the positive horizontal direction is right.
+               See also the constants ``ORIGIN``, ``UP``, ``DOWN``, ``LEFT``,
+               ``RIGHT``, and others, defined in the :mod:`.constants` module.
+
+There are many other possible ways to place mobjects on the screen, for example
+:meth:`.move_to`, :meth:`.next_to`, and :meth:`.align_to`.  The next scene
+``Shapes2`` uses all three.
+
+.. code-block:: python
+
+   class MobjectPlacement(Scene):
+       def construct(self):
+           circle = Circle()
+           square = Square()
+           triangle = Triangle()
+
+           # place the circle two units left from the origin
+           circle.move_to(LEFT * 2)
+           # place the square to the left of the circle
+           square.next_to(circle, LEFT)
+           # align the left border of the triangle to the left border of the circle
+           triangle.align_to(circle, LEFT)
+
+           self.add(circle, square, triangle)
+           self.wait(1)
+
+.. image:: ../_static/building_blocks/placement.png
+    :align: center
+    :alt: figure of a circle, a square, and a triangle
+
+The :meth:`.move_to` method uses absolute units (measured relative to the
+``ORIGIN``), while :meth:`.next_to` uses absolute units (measured from the
+mobject passed as the first argument).  :meth:`align_to` uses ``LEFT`` not as
+measuring units but as a way to determine the border to use for alignment.  The
+coordinates of the borders of a mobject are determined using an imaginary
+bounding box around it.
```

Comment:
Should we somewhere mention the default value for `next_to` is `RIGHT`?

Reply:
Good suggestion!